src/components/PageHero.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { StaticImage } from "gatsby-plugin-image"
+
+class PageHero extends React.Component {
+    render() {
+        const {title, image, subTitle} = this.props
+        return (
+            <div className="page-hero">
+                <StaticImage 
+                        src={ image } 
+                        alt={ title } 
+                        className="page-hero_img"
+                        layout="fixed"
+                />
+                {console.log(image)}
```

Comment:
I would avoid putting console logs in the template like this. Would move this to be right after the const and before the return